ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -139,8 +161,93 @@ public void setPassword(String password) {
 
 	@Override
 	protected void configure() {
+		bind(Session.class).to(DefaultSession.class).in(Scopes.SINGLETON);
```

Comment:
Thats an interesting option, thanks for suggesting it.  Are there any examples of this pattern being used somewhere?  We could certainly give the OptionBinder + factory service a try if it seems preferable.